zsh-syntax-highlighting.zsh

```diff
@@ -1,3 +1,4 @@
+() { # that's here to make the line numbers in errors match the file line numbers.
```

Comment:
This doesn't appear to be true. If I checkout `ca6b31f`, when I open a new terminal I get
```
(anon):46: no such file or directory: /home/phy1729/.version
(anon):47: no such file or directory: /home/phy1729/.revision-hash
zsh-syntax-highlighting: highlighters directory '/home/phy1729/highlighters' not found.
```
 however lines 46-48 are
```
  # $0 is reliable
  typeset -g ZSH_HIGHLIGHT_VERSION=$(<"${0:A:h}"/.version)
  typeset -g ZSH_HIGHLIGHT_REVISION=$(<"${0:A:h}"/.revision-hash)
```
The line numbers seem to be off by one.